### DISCUSSION TITLE
Contact angle of non-wetting phase in `Drainage`/`InvasionPercolation` means

### DISCUSSION THREAD

**Discussion by amisatoayak** [PMEAL/OpenPNM#2629] [Q&A] [answered]
f = op.models.physics.capillary_pressure.washburn
air.add_model(propname='throat.entry_pressure',
              model=f, 
              surface_tension='throat.surface_tension',
              contact_angle='throat.contact_angle',
              diameter='throat.diameter',)

**What does the _contact angle_  of air phase in Drianage/Invasionpercolation algorithms actually mean?
Is it the contact angle of material-wetting(e.g. water) interface α？Or 180-α？**
I personally think it is 180-α, but not sure. Hope someone could give me a certain answer. Thanks.

**Comment by jgostick** [comment, marked as answer]
Hi @amisatoayak 
You have noticed a valid quirk about openpnm. Contact angle is a funny thing since it technically requires 3 phases, yet we only assign it on one phase.  However, to keep things simple, we just put the contact angle for the system on the invading phase, which is the non-wetting phase, so the angle refers to the angle measured through the nonwetting phase like when you have a droplet sitting on a surface.

**Reply by JMCarmona10** [reply]
Regarding contact angles in a two-phase drainage process: Should we only consider the contact angle of the non-wetting (invading) phase, even though it's possible to measure the contact angle of the wetting phase as well? I'm trying to understand if both contact angles are relevant for the drainage process calculations.
